control/trajectory_follower/src/mpc_lateral_controller.cpp

```diff
@@ -338,33 +360,11 @@ MpcLateralController::getInitialControlCommand() const
   return cmd;
 }
 
-bool MpcLateralController::isStoppedState() const
+bool MpcLateralController::isStoppedState()
 {
-  // If the nearest index is not found, return false
-  if (m_current_trajectory_ptr->points.empty()) {
-    return false;
-  }
-
-  // Note: This function used to take into account the distance to the stop line
-  // for the stop state judgement. However, it has been removed since the steering
-  // control was turned off when approaching/exceeding the stop line on a curve or
-  // emergency stop situation and it caused large tracking error.
-  const size_t nearest = motion_utils::findFirstNearestIndexWithSoftConstraints(
-    m_current_trajectory_ptr->points, m_current_kinematic_state_ptr->pose.pose,
-    m_ego_nearest_dist_threshold, m_ego_nearest_yaw_threshold);
-
-  const double current_vel = m_current_kinematic_state_ptr->twist.twist.linear.x;
-  const double target_vel =
-    m_current_trajectory_ptr->points.at(static_cast<size_t>(nearest)).longitudinal_velocity_mps;
-
-  const auto latest_published_cmd = m_ctrl_cmd_prev;  // use prev_cmd as a latest published command
-  if (m_keep_steer_control_until_converged && !isSteerConverged(latest_published_cmd)) {
```

Comment:
It's better to select to whether to control steer when stopping.
`m_keep_steer_control_until_converged` is the option, and we would like to keep it.